Header header
string calibration_status
# in seconds
int32 next_check_countdown
# in seconds
int32 calibration_duration
# current temperature
float32 imu_temperature
# Temperature after last calibration
float32 calibrated_imu_temperature
# is robot moving?
bool robot_moving
